photon-core/src/main/java/org/photonvision/vision/processes/VisionSourceSettables.java

```diff
@@ -62,6 +62,10 @@ public void setVideoModeInternal(int index) {
     }
 
     public void setVideoMode(VideoMode mode) {
+        if (mode.equals(getCurrentVideoMode())) {
```

Comment:
This won't work as VideoMode doesn't implement equals(). Use another comparator instead